custodia.spec

```diff
@@ -0,0 +1,263 @@
+%if 0%{?fedora}
+%global with_python3 1
+%endif
+
+%{!?version: %define version 0.3.1}
+
+Name:           custodia
+Version:        %{version}
+Release:        3%{?dist}
+Summary:        A service to manage, retrieve and store secrets for other processes
+
+License:        GPLv3+
+URL:            https://github.com/latchset/%{name}
+Source0:        https://github.com/latchset/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+Source1:        https://github.com/latchset/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz.sha512sum.txt
+Source2:        custodia.conf
+Source3:        custodia.service
+Source4:        custodia.socket
+Source5:        custodia.tmpfiles.conf
+
+BuildArch:      noarch
+
+BuildRequires:      python2-devel
+BuildRequires:      python-jwcrypto
+BuildRequires:      python2-requests
+BuildRequires:      python2-setuptools >= 18
+BuildRequires:      python2-coverage
+BuildRequires:      python-tox >= 2.3.1
+BuildRequires:      python2-pytest
+BuildRequires:      python2-python-etcd
+BuildRequires:      python-docutils
+BuildRequires:      python2-configparser
+BuildRequires:      python2-systemd
+
+%if 0%{?with_python3}
+BuildRequires:      python3-devel
+BuildRequires:      python3-jwcrypto
+BuildRequires:      python3-requests
+BuildRequires:      python3-setuptools > 18
+BuildRequires:      python3-coverage
+BuildRequires:      python3-tox >= 2.3.1
+BuildRequires:      python3-pytest
+BuildRequires:      python3-python-etcd
+BuildRequires:      python3-docutils
+BuildRequires:      python3-systemd
+%endif
+
+%if 0%{?with_python3}
+Requires:           python3-custodia = %{version}-%{release}
+%else
+Requires:           python2-custodia = %{version}-%{release}
+%endif
+
+# FreeIPA 4.3 and 4.4 are not compatible with custodia because the custodia
+# script now runs under Python 3.
+Conflicts:          freeipa-server-common < 4.5
+Conflicts:          ipa-server-common < 4.5
```

Comment:
custodia >= 0.3 breaks FreeIPA < 4.5 because /usr/bin/custodia uses Python 3 insteand of Python 2. I want to make sure that people notice that there is a conflict.